benchmarks/scripts/expval_benchmark.py

```diff
@@ -21,8 +21,9 @@
 )
 from ucc import compile as ucc_compile
 
-SINGLE_QUBIT_ERROR_RATE = 0.01
-TWO_QUBIT_ERROR_RATE = 0.03
+# error rates representative of current hardware as of Mar 24, 2025
+SINGLE_QUBIT_ERROR_RATE = 0.00052
```

Comment:
Is it possible to add hyperlinks or a reference to the sources used to determine these rates?

Reply:
WDYT of a link to the issue with the data? https://github.com/unitaryfund/ucc/issues/299#issuecomment-2748802174

Reply:
👍